README.md

```diff
@@ -1 +1,157 @@
-ProgressView
+<H1 align="center">ProgressView</H1>
+
+<p align="center">
+<a href="https://cocoapods.org/pods/progress-view"><img alt="Version" src="https://img.shields.io/cocoapods/v/progress-view.svg?style=flat"></a> 
+<a href="https://github.com/Incetro/progress-view/blob/master/LICENSE"><img alt="Liscence" src="https://img.shields.io/cocoapods/l/progress-view.svg?style=flat"></a> 
+<a href="https://developer.apple.com/"><img alt="Platform" src="https://img.shields.io/badge/platform-iOS-green.svg"/></a> 
+<a href="https://developer.apple.com/swift"><img alt="Swift4.2" src="https://img.shields.io/badge/language-Swift5.0-orange.svg"/></a>
+</p>
+
+## Description
+
+This library is a custom progress view.
+ 
+- [Usage](#Usage)
+- [Requirements](#requirements)
+- [Communication](#communication)
+- [Installation](#installation)
+- [Authors](#license)
+
+## Usage <a name="Usage"></a>
+
+To use a *ProgressView*, all you will need to import `ProgressView` module into your swift file:
```

Comment:
To use a *ProgressView*, all you need is to import `ProgressView` module into your swift file: